net/relay.go

```diff
@@ -123,17 +124,23 @@ func (h *Host) SubmitClaimToRelayer(relayerID peer.ID, request *RelayClaimReques
 }
 
 func receiveRelayClaimResponse(stream libp2pnetwork.Stream) (*RelayClaimResponse, error) {
-	msg, err := readStreamMessage(stream, maxRelayMessageSize)
-	if err != nil {
-		return nil, fmt.Errorf("failed to read RelayClaimResponse: %w", err)
-	}
+	const relayResponseTimeout = time.Minute
```

Comment:
Let's say it takes 20 minutes for the contract to get set to to ready, this will give Bob time to try just under 100 relayers assuming they all timeout? I may have the math wrong, that's why I'm asking.

Reply:
what do you mean? bob is only able to try to claim after ready and before t1, which is at minimum an hour right now. I reduced this to 45 seconds which would allow him to try 80 relayers if they're all done sequentially. we can add some concurrency potentially

Reply:
In my example, Bob had 100 minutes until T1, because T0 was still 40 minutes away when the contract was set to ready, plus the additional 60 minutes to reach T1.